protected/humhub/modules/user/models/fieldtype/UserLastLogin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\user\models\fieldtype;
+
+use humhub\libs\Html;
+
+/**
+ * UserLastLogin is a virtual profile field
+ * that displays the user last login dati
+ *
+ * @since 1.6
+ */
+class UserLastLogin extends BaseTypeVirtual
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function getVirtualUserValue($user, $raw = true)
+    {
+        if (empty($user->last_login)) {
+            return '';
+        }
+
+        if ($raw) {
+            return Html::encode($user->last_login);
+        }
+    }
```

Comment:
Missing return statement for non raw output. Please use something like `Yii::$app->formatter->asDate`.

![image](https://user-images.githubusercontent.com/4736168/95206782-65b2f980-07e7-11eb-9927-06ed2166a544.png)

Reply:
Done! @luke-  I did not catch up what is the purpose of the **$raw** parameter? Could you please explain? Thank's.